gdx/src/com/badlogic/gdx/graphics/g2d/Sprite.java

```diff
@@ -529,7 +529,17 @@ public float getX () {
 	public float getY () {
 		return y;
 	}
-
+	
+	/** @return the center x of the sprite, not accounting for scale. */

```

Comment:
Also, what about `originX` and `originY`? This also influences where the center is. If the origin is in the center, `getX()`, `getY()` would give you the center exactly.